services/api-db/docker-entrypoint-initdb.d/00-setup.sql

```diff
@@ -71,6 +71,8 @@ CREATE TABLE IF NOT EXISTS environment (
        deploy_type            ENUM('branch', 'pullrequest', 'promote') NOT NULL,
        environment_type       ENUM('production', 'development') NOT NULL,
        openshift_projectname  varchar(100),
+       lagoon_route           varchar(300),
```

Comment:
If we are adding new columns in the table, we also need an update procedure for existing tables, see
https://github.com/amazeeio/lagoon/blob/f02a574502307efdcc0db9e1ee7a88f595ec6fa2/services/api-db/docker-entrypoint-initdb.d/00-setup.sql#L161-L179
and
https://github.com/amazeeio/lagoon/blob/f02a574502307efdcc0db9e1ee7a88f595ec6fa2/services/api-db/docker-entrypoint-initdb.d/00-setup.sql#L415
as an example